_posts/2021-08-10-Day-2-Post.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: post
+title: "The three Musketeers: Monte Carlo, Monty Python and Lennard Jones"
+author: Usman Jamshed
+---
+
+# Monte Carlo Simulations!
+Traditional integration techniques are very efficient in solving simple integrals, however sometimes integrals can be impoosible to evaluate analytically leading to the use of Monte Carlo simulations. This method uses repeated random sampling to **estimate** the outcomes of an event. Using this technique, impossible to analytically calculate integrals become much more mangeable. However, due to the nature of repeated random sampling a large number of observations needs to be conducted to ensure an accurate estimate. Thus, a balance between speed and accuracy needs to be met, where the simulation is fast (lower number of observations) and accurate (higher number of observations) to the true result.
+
+# Lennard Jones Potentials!
+The Lennard Jones Potential is an approximation of the energy of nonbonded interations (Equation 1). It is a pairwise potential and can be visualized as a graph with two main sections, a repulsive part and an attractive part. Starting far apart, the pair of atoms have negligible interactions and their potenial is ~0, however as you bring them closer together the interaction becomes attactive until a global minimum value of -1 is reached. After this point bringing them any further, causes the interaction to become repulsive with energy potentials reaching infinity as the distance between them approaches 0. Conducting the approximation in reduced units is ideal because using SI units makes the values incredibility inconvinient to work with due to their very large or very small values and the inherent floating point error. The total pair potential energy of a system of particles can also be calculated, by calculating the sum of the Lennard Jones energy over all pairwise interations. 
```

Comment:
```suggestion
The Lennard Jones Potential is an approximation of the energy of nonbonded interactions (Equation 1). It is a pairwise potential and can be visualized as a graph with two main sections, a repulsive domain and an attractive domain. At a relatively great distance, the pair of atoms have negligible interactions and their potential is ~0. However as the particles are brought closer together, the interaction becomes attractive until the point $$r_m$$, where the potential energy is at a minimum of - $\varepsilon$ . At closer distances, the repulsive interactions overtake the attractive forces. The energy potential rising exponentially as the distance between them approaches 0. Conducting the approximation in reduced units is ideal because using SI units makes the values incredibility inconvenient to work with due to their very large or very small values and the inherent floating point error. The total pair potential energy of a system of particles can also be calculated by calculating the sum of the Lennard Jones energies over all pairwise interactions. 
```